python/cryptol/cryptoltypes.py

```diff
@@ -155,7 +155,7 @@ def convert(self, val : Any) -> Any:
             return {'expression': 'bits',
                     'encoding': 'base64',
                     'width': val.length(), # N.B. original length, not padded
-                    'data': copy.get_bitvector_in_hex()}
+                    'data': base64.b64encode(bytes(int(val))).decode('ascii')}
```

Comment:
I'm assuming that you've tested that this works round-trip. The sequence of conversions looks a bit mysterious :-)

Do we still need to pad the copy from the left if we're sending it as base64?

Reply:
I had it wrong. After some googling around, manual experiments, and a better test from the cryptol-remote-api package I think this is right after 90df637.